templates/postun.erb

```diff
@@ -0,0 +1,3 @@
+if [ $1 -eq 0 ]; then
```

Comment:
RPM post uninstall passes arguments?

Reply:
Yes. https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_syntax

Reply:
was news to me too :)